support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -198,43 +185,36 @@ const discountSummaryCopy = (
 };
 
 export function ThreeTierCard({
+	product,
 	cardTier,
 	promoCount,
-	isRecommended,
-	isRecommendedSubdued,
+	isSubdued,
 	isUserSelected,
 	currencyId,
-	countryGroupId,
 	paymentFrequency,
 	link,
 	productDescription,
 	price,
 	promotion,
 	ctaCopy,
-	lozengeText,
-	abParticipations,
+	pillCopy,
 }: ThreeTierCardProps): JSX.Element {
 	const currency = currencies[currencyId];
 	const period = recurringContributionPeriodMap[paymentFrequency];
 	const formattedPrice = simpleFormatAmount(currency, price);
 	const quantumMetricButtonRef = `tier-${cardTier}-button`;
-	const { label, benefits, benefitsSummary, offers, offersSummary } =
-		productDescription;
+	const { title, benefits } = productDescription;
 
 	return (
-		<section
-			css={container(isRecommended, isUserSelected, isRecommendedSubdued)}
-		>
-			{isUserSelected && <ThreeTierLozenge title="Your selection" />}
-			{isRecommended && !isUserSelected && (
-				<ThreeTierLozenge
-					subdue={isRecommendedSubdued}
-					title={
-						promotion?.landingPage?.roundel ?? lozengeText ?? 'Highest impact'
-					}
+		<section css={container(!!pillCopy, isUserSelected, isSubdued)}>
+			{isUserSelected && <ThreeTierCardPill title="Your selection" />}
+			{!!pillCopy && !isUserSelected && (
+				<ThreeTierCardPill
+					subdue={isSubdued}
+					title={promotion?.landingPage?.roundel ?? pillCopy}
```

Comment:
Does this need a fallback if both of these items are `undefined`?

Reply:
We do check that `pillCopy` is defined before rendering the pill.
But actually I've noticed a bug from the [existing implementation](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx#L229) - we don't check if the promo roundel is defined in the condition. I'll fix this here

Reply:
thanks!